### PR TITLE
Author_year citation style

### PR DESCRIPTION
**Pull request by observingClouds**
#55

### CODE REVIEW

File: how_to_eurec4a/intro.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ If you miss some information that could be valuable, feel free to check the link
 ## Useful links
 A list of links to general information about EUREC4A:
 * [official EUREC4A webpage](http://eurec4a.eu/)
-* EUREC4A overview papers:{cite}`Bony:2017` and {cite}`Stevens:2021`
+* EUREC4A overview papers:{cite:t}`Bony:2017` and {cite:t}`Stevens:2021`
```
Comment (d70-t) [inline how_to_eurec4a/intro.md:19]:
```suggestion
* EUREC4A overview papers: {cite:t}`Bony:2017` and {cite:t}`Stevens:2021`
```

**Review by d70-t — COMMENTED** [review]
Thanks for this long-awaited addition :+1:. It looks quite good so far!

As #74 is now merged, there should probably be another merge from `master`, fixing the newly added citations. Afterwards, this PR would be good to go.

As a tiny visual improvement, I'm wondering if we should try to make the brackets round, using a method like the one described [here](https://sphinxcontrib-bibtex.readthedocs.io/en/2.4.1/usage.html?highlight=author_year#custom-inline-citation-references). Due to jupyterbook, this might also require a local extension as described [here](https://jupyterbook.org/advanced/sphinx.html#local-sphinx-extensions). As this is quite a bit of work for "only" a little visual improvement, I'd also be fine to make this a separate issue and merge this PR in its current form.

**Comment by d70-t** [comment]
As mentioned in #55, this PR should probably update to `jupyter-book>=0.12.2` in `requirements.txt`.

File: how_to_eurec4a/intro.md (hunk 1 above)
Comment (observingClouds) [reply how_to_eurec4a/intro.md]: Done!

**Comment by observingClouds** [comment]
The updated pull-request includes now the visual improvements as well:
- round brackets
- improved references by using the APA style.
  Before:
  image

  After:
  image

**Comment by observingClouds** [comment]
Ah, one moment. It seems like the APA references did not work properly in the test. I must have forgotten to push something.

**Comment by d70-t** [comment]
This would be an awesome solution! However, in the current build, I only see the following...
APAa

**Comment by observingClouds** [comment]
Now it should work @d70-t. The build looks promising. It just timed out for the cloud mask page.

**Review by d70-t — APPROVED** [review]
Thanks for ths PR, this citation style is **much** better than what we've had before!

**Comment by d70-t** [comment]
I really would like to know what's the issue behind the cloudmasks article... But I've no idea yet...

**Pull request merged**